charts/operator-wandb/local-development.md

```
@@ -149,18 +143,12 @@ To deploy a different test configuration, update the `values` field in your `til
 ```
 
 ### 5. Accessing the Deployed Services
 
-Tilt automatically sets up port forwarding for the deployed services. By default:
-
-- App service: http://localhost:8080
-- Console service: http://localhost:8082
-- Weave-trace service (if enabled): http://localhost:8722
-
-You can customize these ports in your `tilt-settings.json` file.
+Tilt automatically sets up port forwarding to the services via an nginx proxy at http://localhost:8080.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)